gulpfile.js

```diff
@@ -26,40 +26,63 @@ var browserifyOptions = {
     bundleExternal: false
 };
 
-gulp.task('bower', function(cb){
-    bower.commands.install().on('end', function (installed){
+gulp.task('bower', function(cb) {
+    bower.commands.install().on('end', function(installed) {
         console.log(installed);
         cb();
     });
 });
 
 gulp.task('clean', ['lint'], function(cb) {
-    del([ DEST ], cb);
+    del([DEST], cb);
 });
 
-gulp.task('lint', function(){
+gulp.task('lint', function() {
     return gulp.src(['./*.js', './lib/*.js'])
         .pipe(jshint())
         .pipe(jshint.reporter('default'));
 });
 
-gulp.task('build', ['clean'], function () {
+gulp.task('clean-doc', function(cb) {
+    del(['doc'], cb);
+});
+
+gulp.task('jsdoc', ['clean-doc'], function() {
+    var template = {
+        path: 'ink-docstrap',
+        systemName: pjson.name,
+        footer: '',
+        copyright: 'Ethereum',
+        navType: 'vertical',
+        theme: 'cerulean',
+        linenums: true,
+        collapseSymbols: false,
+        inverseNav: false
+    };
+
+    gulp.src(['./lib/*.js', 'README.md', './package.json'])
```

Comment:
that's really cool, but jsdoc output is right now incomplete. it only outputs 'readme' && 'web3'. Do you how can we make this better? You don't have to fix this in this pr.

and also shouldn't this line be? :
```javascript
    gulp.src(['./lib/**/*.js', 'README.md', './package.json'])
```